x-pack/test/security_solution_api_integration/test_suites/entity_analytics/risk_engine/trial_license_complete_tier/asset_criticality.ts

```diff
@@ -202,7 +202,7 @@ export default ({ getService }: FtrProviderContext) => {
         enableAssetCriticalityAdvancedSetting(kibanaServer, log);
       });
 
-      it('should return the first 10 asset criticality records if no args provided', async () => {
+      it('@skipInServerless should return the first 10 asset criticality records if no args provided', async () => {
```

Comment:
This test was not executed in the PR pipelines because it is in the `@skipInServerlessMKI` group. Now that it has been enabled, it is failing and needs to be fixed.

Reply:
Entity Analytics follow-up issue [here](https://github.com/elastic/kibana/issues/189067).